src/main/java/core/basesyntax/converter/ConverterCSVToObject.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.converter;
+
+import com.opencsv.bean.CsvToBeanBuilder;
+import core.basesyntax.entity.FruitTransaction;
+
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.util.List;
+
+public class ConverterCSVToObject {
+
+    public List<FruitTransaction> reader(String inputFileName) throws FileNotFoundException {
+        List<FruitTransaction> fruitTransactionList;
+        fruitTransactionList = new CsvToBeanBuilder<FruitTransaction>(new FileReader(inputFileName))
```

Comment:
do return here inline, no need to create variable

Reply:
@al-pavlenko done